rsconnect/main.py

```diff
@@ -666,6 +666,9 @@ def add(
     set_verbosity(verbose)
     output_params(ctx, locals().items())
 
+    if not name:
```

Comment:
We can't directly make `--name` required in `@server_args` because that's used elsewhere. It would add some annoying redundancy to create a second version of it with `--name` set to required, so this seemed like the simplest solution